tests/testthat/test-deploy-pushes-to-sql-server.R

```diff
@@ -18,6 +18,14 @@ context("Checking deploy predictions from sql server to sql server")
   ,[ThirtyDayReadmitFLG]
   FROM [SAM].[dbo].[HCRDiabetesClinical]
   "
+
+#### BEGIN TESTS ####
+
+ test_that("LMM deploy classification pushes values to SQL", {
+
+  skip_on_travis()
+  skip_on_cran()
+
   df <- selectData(connectionString, query)
```

Comment:
This will change as #402 implemented, but it fixes the build now. That is more of a long term change anyways.